nva-datamodel-java/src/main/java/no/unit/nva/model/instancetypes/artistic/performingarts/PerformingArtsSubtype.java

```diff
@@ -0,0 +1,59 @@
+package no.unit.nva.model.instancetypes.artistic.performingarts;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import nva.commons.core.JacocoGenerated;
+
+import java.util.Objects;
+
+public class PerformingArtsSubtype {
+    public static final String TYPE = "type";
+    public static final String DESCRIPTION = "description";
+    @JsonProperty(TYPE)
```

Comment:
To my understanding, this is a way to overcome some feature that Jackson does not cover. Can we indicate this somehow? We could for example create an interface with a name "VariablyTyped" that would expect `getType` to be implemented and would also contain a Javadoc comment explaining the role of this interface. The reaason I am saying this, is to avoid the temptation of changing the field name, or if we ever use another serialization library, to maybe address the problem differently.